release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -86,6 +86,39 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 
 In {product-title} 4.12, the External DNS Operator modifies the format of the ExternalDNS wildcard TXT records on AzureDNS. The External DNS Operator replaces the asterisk with `any` in ExternalDNS wildcard TXT records. You must avoid the ExternalDNS wildcard A and CNAME records having `any` leftmost subdomain because this might cause a conflict.
 
+[id="ocp-4-12-nw-metrics-telemetry"]
+==== Capturing metrics and telemetry associated with the use of routes and shards
+
+In {product-title} 4.12, the Cluster Ingress Operator exports a new metric named `route_metrics_controller_routes_per_shard`. The `name` label of the metric specifies the name of the shards. This metric gives the total number of routes that are admitted by each shard. The following metrics are sent to {product-title} by using telemetry.
```

Comment:
@xenolinux you missed the suggestion for the last line as per this comment (https://github.com/openshift/openshift-docs/pull/51555#discussion_r997067884)

```suggestion
In {product-title} 4.12, the Cluster Ingress Operator exports a new metric named `route_metrics_controller_routes_per_shard`. The `name` label of the metric specifies the name of the shards. This metric gives the total number of routes that are admitted by each shard.

The following metrics are sent through telemetry.
```

Reply:
acked